lib/modules/model/types/ModelDefinition.ts

```diff
@@ -18,6 +17,34 @@ import { Decoder, NamedMeasures } from "../../../modules/decoder";
  *     },
  *     defaultMetadata: {
  *       height: 20
+ *     },
+ *     metadataDetails: {
+ *       "extTemp": {
+ *         "group": "buildingEnv",
+ *         "locales": {
+ *           "en": {
+ *             "groupFriendlyName": "Building environment",
+ *             "friendlyName": "External temperature",
+ *             "description": "Container external temperature"
+ *           },
+ *           "fr": {
+ *             "friendlyName": "Température externe",
```

Comment:
French translation does not have a groupFriendlyName

Reply:
it's an error, groupFriendlyName is only present in metadataGroups object I removed all errors